packages/ui/src/components/editor/properties/media/index.tsx

```diff
@@ -99,9 +100,9 @@ export const MediaNodeEditor: EditorComponentType = (props) => {
           min={0}
           max={100}
```

Comment:
wouldn't it be better to change the min/max/step ?

Reply:
the min max is for the slider UI, is we want the values to be between 1 - 100, we need to use a converter

Reply:
we can keep the values between 0 - 2 as well, if needed, is that preffered?